README.md

```diff
@@ -86,6 +93,12 @@ nix profile list | grep ocm
 ocm --help
 ```
 
+### Install from AUR (Arch Linux User Repository)
+
+git-urL: https://aur.archlinux.org/ocm-cli.git
+
+[AUR Documentation](https://wiki.archlinux.org/title/Arch_User_Repository)
+
```

Comment:
can you please document the rudimentary how to steps on installing the package? referencing to the AUR Documentation is in imho good, but not sufficient.
also, i found this link https://aur.archlinux.org/packages/ocm-cli that also includes a reference to which version is current.

Reply:
@vasu1124 : done
as for the page-link: I added this earlier above (line 59). I can of course repeat it here 🤷🏻‍♂️ 
-> done

Reply:
on another note: if someone would be interested to include updating into release-pipeline, I can offer to grant required privileges to pipeline-user. I already created a shellscript for doing the updateing, so integration should be quite easy

Reply:
For sure we should have the upload of the latest released package to all mentioned package managers in the Github action "Release". I opened https://github.com/open-component-model/ocm/issues/856 to track that.